### ISSUE TITLE
meta page picker doesn't seem to work

### ISSUE THREAD

**Issue by Rokainu-Codes** [silverbulletmd/silverbullet#985]
Whenever I try to open the meta page picker nothing happens and I get this error message in the console:
![image](https://github.com/user-attachments/assets/e305542d-e22d-4df0-b168-cedf0fd6a829)

**Comment by Rokainu-Codes** [comment]
This seems to work fine in a fresh silverbullet, so this may be an issue with one of my files.

**Comment by zefhemel** [comment]
This was a bug that should have been fixed in 0.8.4. It happened when you have broken links in your space. Could that be if?

**Comment by Rokainu-Codes** [comment]
I updated my version to 0.8.4, though that didn't fix it yet. After removing all the broken links in my space it started working again.

**Comment by zefhemel** [comment]
Are you still seeing this issue? Just released 0.8.5.

**Comment by Rokainu-Codes** [comment]
I only tested it on mobile so far, it does open the meta picker now but then I can't actually select any of the files. Also all other commands stop working after using it until I delete my cookies. Will test it on pc later as well.

**Comment by zefhemel** [comment]
It would be good to get any sense of errors you're still seeing (harder to extract from mobile, I know). I'm testing this on mobile too (and I have plenty of broken links) and all seems fine.

**Comment by Rokainu-Codes** [comment]
After trying some things, it turned out that I only had this problem using Vivaldi as web browser. Also, re-installing Vivaldi fixed the issue.

I did eventually manage to get a console on mobile, this was the error I kept getting (but not anymore after re-installing Vivaldi):
![Screenshot_20240731-121948_Vivaldi](https://github.com/user-attachments/assets/6c064eba-a040-4834-98c5-d96e98aa1f31)

Not sure why this happened, but it seems 0.8.5 fixed it anyway. Thanks!
